scipy/stats/_stats_py.py

```diff
@@ -6463,7 +6476,13 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
 _ttest_ind_dep_msg = "Use ``method`` to perform a permutation test."
 
 
-@xp_capabilities(cpu_only=True, exceptions=["cupy", "jax.numpy"])
```

Comment:
out of scope: we have no means of testing that jax isn't actually quietly dragging everything from GPU memory to CPU memory and converting to numpy because of https://github.com/jax-ml/jax/issues/25998